Automattic/wp-calypso
# Untangling: Apply new card design in Settings -> Administration (first layer)

Related to p9Jlb4-eSF-p2

## Proposed Changes

This PR applies the new card design in the first layer of Settings -> Administration subtab:

|Simple site|
|-|
|<img width="500" alt="image" src="https://github.com/user-attachments/assets/06732ae1-7c51-4002-8aaa-b2ebbd105298">|

|Atomic site|
|-|
|<img width="500" alt="image" src="https://github.com/user-attachments/assets/dedecca7-7318-4570-82da-360eaf9a1c93">|

|Jetpack site|
|-|
|<img width="500" alt="image" src="https://github.com/user-attachments/assets/14c3fb3e-910c-4ba6-b5c1-5565ba123768">|



## Why are these changes being made?

pbxlJb-6ye-p2

## Testing Instructions

For each {Simple, Atomic, Jetpack} site:

1. Open `/sites/settings/administration/:slug` and verify you can see the cards as shown above.
   - Click the cards and verify that the following cards open in-line in the global view:
      - Reset site
      - Transfer site
      - Delete your site permanently
      - Manage connection
1. Open `/settings/general/:slug` and verify you can see the same set of items.
   - Click the cards and verify that the above cards open in the old Calypso single-site view.

> [!NOTE]  
> The back button and the card layout in each card in the "inner page" still follow the existing production behavior -- will follow up later.

> [!WARNING]  
> The "Copy site" card takes a bit delay to appear...

## Pre-merge Checklist
